Sources/Shared/Models/ConnectedApp.swift

```diff
@@ -182,20 +182,20 @@ public class ConnectedApp: VIMModelObject {
 
     public override func getObjectMapping() -> Any? {
         return [
-            String.Key.addDate: String.Value.addDate,
-            String.Key.dataAccessIsExpired: String.Value.dataAccessIsExpired,
-            String.Key.publishCategories: String.Value.publishCategories,
-            String.Key.thirdPartyUserID: String.Value.thirdPartyUserID,
-            String.Key.thirdPartyUserDisplayName: String.Value.thirdPartyUserDisplayName,
-            String.Key.type: String.Value.type
+            Constants.Key.addDate: Constants.Value.addDate,
+            Constants.Key.dataAccessIsExpired: Constants.Value.dataAccessIsExpired,
+            Constants.Key.publishCategories: Constants.Value.publishCategories,
+            Constants.Key.thirdPartyUserID: Constants.Value.thirdPartyUserID,
+            Constants.Key.thirdPartyUserDisplayName: Constants.Value.thirdPartyUserDisplayName,
+            Constants.Key.type: Constants.Value.type
```

Comment:
@jasonhawkins what is we used tuples to store the constants for the key-value pairs so we could facilitate the correct pairing? For example

```swift
static let addDate = (key: "add_date", value: "addDateString")
```
would mean we could do 

```swift
[Constants.addDate.key: Constants.addDate.value]
```
And if you left of the key words you could make it even shorter as 

```swift
[Constants.addDate.0: Constants.addDate.1]
```

Reply:
Ohh nice. It'll work for things that are 1:1 key-value pairs, like mapping properties in an object. But for cases where we have a key that maps to a class, the value is derived from the name of the class and not an explicitly defined String: 

```Swift
public override func getClassForObjectKey(_ key: String?) -> AnyClass? {
    switch key {
    case Constants.Key.publishBlockers:
        return PublishJobBlockers.self
    default:
        return nil
    }
}
```

I'll use this approach for the 1:1 pairs, and move the orphans into the private String extensions.